playwright/user-stories/view-items.md

```diff
@@ -8,3 +8,71 @@
 Given there are multiple volumes in an item
 When the researcher views the item
 Then the volumes should be browsable
+
+Title: A user I can view an item page
+
+As a researcher
```

Comment:
If we follow the pattern of [this article](https://dannorth.net/whats-in-a-story/) - the title and narrative have already been set. We're fleshing out the scenarios. 

We can move the current scenario down a bit, and maybe change the narrative to more what you have here - but I don't think we need another narrative.